apps/frontend/src/app/5_pages/MarketMakingPage/components/AdjustAndDepositModal/AdjustAndDepositModal.utils.ts

```diff
@@ -0,0 +1,12 @@
+import { Decimal } from '@sovryn/utils';
+
+import { DEFAULT_AMM_SLIPPAGE } from './AdjustAndDepositModal.constants';
+
+export const getMinReturn = (amount: Decimal) =>
+  amount.sub(amount.mul(DEFAULT_AMM_SLIPPAGE).div(100)).toNumber().toFixed(0);
```

Comment:
```suggestion
  amount.sub(amount.mul(DEFAULT_AMM_SLIPPAGE).div(100)).toString(0);
```
I think you can just use `toString(0)`.